code/part-one/blockchain.js

```diff
@@ -18,13 +18,13 @@ class Block {
       - transactions
       - the previous block's hash
   */
- constructor(timestamp, transactions, previousHash = '') {
-  this.timestamp = timestamp;
-  this.transactions = transactions;
-  this.previousHash = previousHash;
-  this.hash = this.calculateHash();
-  this._nonce = 0;
-}
+  constructor(timestamp, transactions, previousHash = '') {
+    this.timestamp = timestamp;
+    this.transactions = transactions;
+    this.previousHash = previousHash;
```

Comment:
These whitespace additions confuse the diff a bit. Would prefer they be rebased into the appropriate previous commits.